File: src/main/java/io/github/toolsjs/kubejs/builders/ShieldItemBuilder.java
package io.github.toolsjs.kubejs.builders;

import dev.latvian.mods.kubejs.item.custom.HandheldItemBuilder;
import net.minecraft.resources.ResourceLocation;
import net.minecraft.world.item.Item;
import net.minecraft.world.item.ShieldItem;

public class ShieldItemBuilder extends HandheldItemBuilder {


    public ShieldItemBuilder(ResourceLocation i) {
        super(i, 0, 0);
    }

    @Override
    public Item createObject() {
        return new ShieldItem(createItemProperties());
    }
}

File: src/main/kotlin/io/github/toolsjs/kubejs/builders/ShieldItemBuilder.kt
package io.github.toolsjs.kubejs.builders

import dev.latvian.mods.kubejs.item.custom.HandheldItemBuilder
import net.minecraft.resources.ResourceLocation
import net.minecraft.world.item.Item
import net.minecraft.world.item.ShieldItem

class ShieldItemBuilder(i: ResourceLocation?) : HandheldItemBuilder(i, 0f, 0f) {
    override fun createObject(): Item {
        return ShieldItem(createItemProperties())
    }
}
